src/components/header/header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { StyledHeader } from "./header.styles";
+
+const Header = ({ text, back }) => {
```

Comment:
yeah you can also remove the back because we aren't needing to pass that in here, but pass in a `subtext` variable instead!